pixi.toml

```diff
@@ -36,12 +36,10 @@ obelisk_py = { path="./obelisk/python", editable=true }
 
 # cuda
 [feature.cuda121]
-channels = ["nvidia", "nvidia/label/cuda-12.1.0"]
-system-requirements = {cuda="12.1.0"}
+system-requirements = {cuda="12.1.1"}
 
 [feature.cuda121.dependencies]
-cuda = { version="*", channel="nvidia/label/cuda-12.1.0" }
-
```

Comment:
What is the idea behind removing these channels?

Reply:
great question. for some reason, i noticed that having those channels made all the downstream cuda deps much higher versions like 12.5 (this causes problems for zed). if i removed the channels, then suddenly everything installed correctly via pixi. you'll notice that if you do `pixi list | grep cuda` now, all the versions of everything are `12.1.*`, which is what we want